libbeat/processors/dns/config.go

```diff
@@ -120,6 +120,9 @@ type cacheConfig struct {
 
 // cacheSettings define the caching behavior for an individual cache.
 type cacheSettings struct {
+	// Disable the use of the cache.
+	Enabled bool `config:"enabled" default:"true"`
```

Comment:
What uses this `default` struct tag? I don't know of anything using this, but if there is, let me know. 

Otherwise, let's remove this struct tag and update the godoc for the field to document that the default is true.

Reply:
Good one, while examples are in the codebase since 2020 https://github.com/elastic/beats/blob/d3ef76b5fb792a1041d48c117af6e3eaed0352a2/x-pack/filebeat/input/awscloudwatch/config.go#L22 
I guess there's nothing actually making use of this annotation, goland added so convincingly.

Let me remove it.